src/routing/link.tsx

```diff
@@ -24,6 +24,9 @@ const Link: React.StatelessComponent<LinkProps> = ({
   const click = e => {
     onClick(e);
 
+    // Use the default link behavior if the slashes separating the scheme are present
+    if (to.includes('://')) { return; }
```

Comment:
I don't think this is right. `/my/://weird/url` should not trigger this behavior.